packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -155,11 +176,14 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
     }
 
     fun initialize(config: DatadogFlutterConfiguration) {
-        val configuration = config.toSdkConfiguration()
+        val configurationBuilder = config.toSdkConfiguration()
         val credentials = config.toCredentials()
 
+        _InternalProxy.setTelemetryConfigurationEventMapper(configurationBuilder,
+            FlutterConfigurationTelemetryEventMapper(this))
```

Comment:
minor: we can simply use anonymous class

```suggestion
        _InternalProxy.setTelemetryConfigurationEventMapper(configurationBuilder,
            object : EventMapper<TelemetryConfigurationEvent> {
                override fun map(event: TelemetryConfigurationEvent): TelemetryConfigurationEvent? {
                    // do stuff
                }
            })
```

Reply:
My Kotlin is not so good for this -- how do I access the members of DatadogSdkPlugin? Doesn't `this` refer to that anonymous class in that case?

Reply:
yes, it will refer to anonymous class, but anyway you have full access to the members of the outer class as well.